src/app/signalen/samenvatting/page.tsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+'use client'
+
+import {
+  Heading,
+  Paragraph,
+  Column,
+  Button,
+  Link as ADSLink,
+  Grid,
+  VisuallyHidden,
+} from '@amsterdam/design-system-react'
+import { useRouter } from 'next/navigation'
+import { useForm } from 'react-hook-form'
+import Link from 'next/link'
+import { BackLink } from '../_components/BackLink'
+import { useFormContext } from '../FormContext'
+
+import '../_components/SummaryDescriptionList/summary-description-list.css'
+import './edit-link.css'
+import { capitalizeFirstLetter } from '../_utils/capitalizeFirstLetter'
```

Comment:
There CSS for this which should not make any difference to screen reader I think.

Reply:
True, but I don't want to capitalize all fields. The email field isn't capitalized. You could do the same with an extra class, but this was easier.